easybuild/easyconfigs/p/PyStan/PyStan-2.19.1.1-foss-2020b.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'PythonPackage'
```

Comment:
Please make this a PythonBundle. We're trying to move in that direction for all Python packages when possible.

Reply:
Moving to PythonBundle is good news, will make tools like easy_update operate better.